### PR TITLE
Fix/documentation clarification

### PR DESCRIPTION
**Pull request by solid-akb**
Small updates to documentation:

There were a few spots where the argument is slightly off or maybe hadn't been updated. The Type is correct, just updating the name for documentation purposes.

### CODE REVIEW

**Review by ajacksified — DISMISSED** [review]
The consistency of the improved parameter names is nice!

File: src/resource/file.ts [hunk 1]
```diff
@@ -85,14 +85,14 @@ function containsReserved(header: Record<string, string>): boolean {
  * @returns The file as a blob.
  */
 export async function getFile(
-  input: Url | UrlString,
+  fileUrl: Url | UrlString,
```
[the remark this answers is not in the document]
Comment (solid-akb) [reply src/resource/file.ts:88]: Done

**Review by ThisIsMissEm — CHANGES_REQUESTED** [review]
The changes aren't quite correct, as all these methods can accept a URL or a SolidDataset (with resource info), what I'd recommend doing is just updating the typedocs so the parameter name matches the argument name, but leave the implementation as-is.

**Review by ThisIsMissEm — APPROVED** [review]

**Comment by ThisIsMissEm** [comment]
@solid-akb I've rebased this, and flattened it into a single commit, whilst keeping you as the author (I used your GitHub noreply as the author address)

**Pull request merged**